templates/XStepExplanation/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Centered from '../chromes/Centered'
+import * as List from '../../List'
+
+import classNamesBind from 'classnames/bind'
+import defaultStyles from './styles.scss'
+
+const baseClass = 'three-step-explanation'
+
+const classes = {
+  list: `${baseClass}__list`
+}
+
+const classNames = classNamesBind.bind(defaultStyles)
+
+export default function XStepExplanation ({
```

Comment:
We have a help thing that looks very different (and better, I would say) on KDS. Maybe the designers are not in sync about that, I donno. 

<img width="573" alt="screen shot 2016-10-13 at 5 34 48 pm" src="https://cloud.githubusercontent.com/assets/308196/19353238/574b9434-916b-11e6-9d36-53b5f6a7cdf2.png">
<img width="587" alt="screen shot 2016-10-13 at 5 34 40 pm" src="https://cloud.githubusercontent.com/assets/308196/19353237/574b307a-916b-11e6-9dea-4e35b079a77d.png">
<img width="613" alt="screen shot 2016-10-13 at 5 34 32 pm" src="https://cloud.githubusercontent.com/assets/308196/19353239/57525f76-916b-11e6-87ee-403f5af94a45.png">